src/p2p/net_node.h

```diff
@@ -63,6 +63,7 @@ namespace nodetool
   {
     peerid_type peer_id;
     uint32_t support_flags;
+    bool m_in_timedsync;
```

Comment:
Might want to add a constructor for this type. Marking the default constructor as deleted will also work, but I think gcc 4.9 incorrectly complains about that technique. The _potential_ issue is the variable was previously being initialized in the constructor, but will no longer be initialized when the default constructor is used for this other type.